### DISCUSSION TITLE
[Support]: what that icon means?

### DISCUSSION THREAD

**Discussion by MG-Sky** [blakeblackshear/frigate#13392] [General Support]
### Describe the problem you are having

![image](https://github.com/user-attachments/assets/70bf3f7d-ba03-4c97-b9c4-83cfef7846c3)
i have one event with that icon but cannot find what it is.

### Version

0-14

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
Na
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wireless

### Camera make and model

reolink

**Comment by banthungprong** [comment]
Move your mouse above the icon and it should show a text info
![image](https://github.com/user-attachments/assets/7e8668d7-9121-4258-82ea-056664187080)

I just searched in the sources. It is a react Icon "LuLassoSelect" and is used as default in IconUtiltsx when nothing was found in the case statements before. So it is some "unknown" object.

    default:
      return <LuLassoSelect key={label} className={className} />;

**Reply by MG-Sky** [reply]
i t was hard to move mouse on the phone :)

but on computer it was 'Phone'

**Comment by banthungprong** [comment]
I wanted to know what Icons are possible. So I searched and saved them. Might be useful for other users too. My favorite ist the mouse. Never saw this one on the road before

- "bear"  GiPolarBear
![image](https://github.com/user-attachments/assets/825bcffb-c3d2-4423-8c4d-2e1725b7b56b)
 
- "bicycle"  FaBicycle 
![image](https://github.com/user-attachments/assets/d3259b40-09e0-4852-834b-7f98a9c1ceab)

- "bird"  GiHummingbird
![image](https://github.com/user-attachments/assets/2ed845f4-be3b-47a0-a95b-818e351bb976)

- "boat"  GiSailboat
![image](https://github.com/user-attachments/assets/27b8d9b7-a434-49a6-a1c7-16fa15c25ee2)

- "bus"  FaBus
![image](https://github.com/user-attachments/assets/5b7f3a91-fa6c-4bdf-a9ec-87777d7bf04a)

- "car""vehicle"  FaCarSide
![image](https://github.com/user-attachments/assets/59abb3b3-dad6-4237-a5bb-113d4a47013f)

- "cat"  FaCat
![image](https://github.com/user-attachments/assets/4c770b05-c3ed-4255-ae76-e0bcde984eec)

- "deer"  GiDeer
![image](https://github.com/user-attachments/assets/beb48016-a210-4b96-8b08-5efda18905cf)

- "animal""bark""dog"  FaDog
![image](https://github.com/user-attachments/assets/7220f4e6-0f47-4316-8204-17ccce6d8047)

- "fire_alarm"  FaFire
![image](https://github.com/user-attachments/assets/c1aa580f-d26a-4434-90ba-cb766af458a8)

- "motorcycle"  FaMotorcycle
![image](https://github.com/user-attachments/assets/72bfb982-b75f-4696-88d8-48ddbbfe0540)

- "mouse"  FaMouse
![image](https://github.com/user-attachments/assets/27866eda-4bc1-4317-8226-6b6a3fcb037d)

- "package"  LuBox
![image](https://github.com/user-attachments/assets/f25ac88e-a173-4d14-8ead-bd38825c53ee)

- "person"  BsPersonWalking
![image](https://github.com/user-attachments/assets/2bfd04e9-e519-43b1-945e-817751b79cf2)

- "sports_ball"  FaFootballBall
- // audio
- "crying""laughter""scream""speech""yell"  MdRecordVoiceOver
![image](https://github.com/user-attachments/assets/60dec499-7da0-4ea9-8161-103deb9482a5)

- // sub labels
- "amazon"  FaAmazon 
![image](https://github.com/user-attachments/assets/a9c3171d-1e55-4f8a-9ef6-ec26d2bb38c1)

- "fedex"  FaFedex
![image](https://github.com/user-attachments/assets/cf88bbe0-f4bd-4015-9fb9-0f46b64491b2)

- "ups"  FaUps
![image](https://github.com/user-attachments/assets/edd983b1-80f3-497f-8ab2-0878a02f94e8)

- "usps"  FaUsps
![image](https://github.com/user-attachments/assets/7d2f1370-cd14-4dc1-ba2b-7cd31f2ec3a7)

- "default" LuLassoSelect
![image](https://github.com/user-attachments/assets/7dbc76e3-0b23-4faa-8562-d205b42a3604)

**Reply by hawkeye217** [reply]
The `mouse` in the default COCO model Frigate uses is actually a computer mouse, not a rodent :)
